src/testers.jl

```diff
@@ -44,6 +39,29 @@ function _make_fdm_call(fdm, f, ȳ, xs, ignores)
         @assert length(callargs) == length(xs)
         return f(callargs...)
     end
+    return fnew
+end
+
+@deprecate _make_fdm_call(fdm, f, ȳ, xs, ignores) _make_j′vp_call(fdm, f, ȳ, xs, ignores)
+
+"""
+    _make_j′vp_call(fdm, f, ȳ, xs, ignores) -> Tuple
+
+Call `FiniteDifferences.j′vp`, with the option to ignore certain `xs`.
+
+# Arguments
+- `fdm::FiniteDifferenceMethod`: How to numerically differentiate `f`.
+- `f`: The function to differentiate.
+- `ȳ`: adjoint w.r.t. output of `f`.
+- `xs`: Inputs to `f`, such that `y = f(xs...)`.
+- `ignores`: Collection of `Bool`s, the same length as `xs`.
+  If `ignores[i] === true`, then `xs[i]` is ignored; `∂xs[i] === nothing`.
+
+# Returns
+- `∂xs::Tuple`: Derivatives estimated by finite differencing.
```

Comment:
Should we start calling it ` x̄s`, or would that be confusing because it is being estimated via finite differencing?

```suggestion
- ` x̄s::Tuple`: Derivatives estimated by finite differencing.
```

Reply:
Currently the notation here is a bit of a Frankenstein. `∂xs` is more consistent with our documented notation, but then the input would need to be `Δy` not `ȳ`. As this docstring is just copied, I'd prefer notation changes be decided in a later PR.